src/node.js

```diff
@@ -393,15 +399,22 @@
     function TickObject(c) {
       this.callback = c;
       this.domain = process.domain || null;
+      this._args = undefined;
     }
 
+    TickObject.prototype._setArgs = function _setArgs() {
+      this._args = arguments;
```

Comment:
Isn't using the arguments object like this a de-opt?

i.e. https://github.com/petkaantonov/bluebird/wiki/Optimization-killers#32-leaking-arguments